### ISSUE TITLE
looks like they couldnt handle the crisnonsensexd style

### ISSUE THREAD

**Issue by DisconnectedDueToInactivity** [CornetPanique86/xxlsteve.net#1]
ENGLISH

Discord chat rules :

1. Do not flood
2. Do not spam
3. Do not send role mentions, @everyone or @here
4. Do not write bot commands except in the channel 🤖》bot-commands 
(music commands in 🎵》music-commands  )
5. No advertising except in 📬》advertisements 
6. Be kind, do not insult others!
7. Please, do not use bad words!
8. Go Live or streams in the vocal channel 📺》Go Live/Stream
9. Respect the rules written in each channel's description/subject!
10. Don't ghost ping

Advanced rules :

10. NSFW messages (messages non adapted for minors) are forbidden!
11. Inappropriate links are forbidden!
12. Inappropriate language is forbidden!
13. Inappropriate Discord profile pics are forbidden!
14. Religious/racious* disputes are forbidden!
15. (cyber) bullying is forbidden!
16. When using the voice channels, avoid obnoxious behaviour such as screaming, blasting music, playing disturbing/loud sounds in the Music channel, etc. Also, please avoid using annoying voice changers or soundboards (but they are both allowed especially if it's for your own privacy)

*This also includes any other arguments/disputes in chat that staff members deem worthy of punishment. In arguments the oppressor will be banned. For example, in a racial argument the player(s) being hateful towards the race will be banned. However the player(s) defending the race will not.

In a nutshell :
You don't need to read all the rules, here are the rules in a nutshell :
Respect the rules by default on all Discord servers. Only read the rules 4 & 5.

🔼 Especially read "In a nutshell" for a fast knowing of the rules! 🔼 

[ @everyone  ]

**Comment by CornetPanique86** [comment]
I was so confused and surprised cuz I thought someone actually looked at my code and wanted to fix it or something lol. crisnonsensexd style has been handled and works now, issue closed 👍🏼
![hog-boar-emoji-mindblown-mind-blown-slideshow-meme](https://user-images.githubusercontent.com/80220423/208975584-af968934-dd78-4d0e-8ae9-fa0559dfd7e8.gif)
